model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -79,6 +79,12 @@ data class OrtConfiguration(
      */
     val licenseFilePatterns: LicenseFilePatterns = LicenseFilePatterns.DEFAULT,
 
+    /**
+     * The package configuration providers to use. Does not define a default, because this is done by the
+     * EvaluatorCommand's `packageConfigurationsDir` and `packageConfigurationsFile` CLI options.
```

Comment:
This seems like a weird rationale to me to not define a default, esp. when comparing to `packageCurationProviders` below. Do you plan to make a follow-up that removes the evaluator's CLI options in favor of these config options?

Reply:
Note: `orthw` requires that it is possible to override `packageConfigurations` and `packageCurations` in the evaluator.
           I believe there should be a way which is simpler than generating a `config.yml` for ORT.

Reply:
> Do you plan to make a follow-up that removes the evaluator's CLI options in favor of these config options?

Yes I thought about that, but didn't want to break backwards compatibility for this initial introduction of the package-configuration plugin system.
The `orthw` requirement is another good reason to keep the CLI option for now.

Reply:
> if the evaluator command options are set, they should override the providers from the configuration file.

Yes, I think from a user's perspective that makes the most sense.